reframe/core/pipeline.py

```diff
@@ -158,7 +158,7 @@ class RegressionTest(metaclass=RegressionTestMeta):
                                             typ.List[str])
 
     #: List of systems supported by this test.
-    #: The general syntax for systems is ``<sysname>[:<partname]``.
+    #: The general syntax for systems is ``<sysname|*>[:<partname]``.
```

Comment:
Better keep the `sysname` reference here as before and add a separate sentence explaining what `sysname` and `partname` could be. I am saying  also `partname` here, because I think it'd be better if all of the following were supported:

- `*:*`: equivalent to `*`
- `*:gpu`: what you implement in this PR
- `system:*`: equivalent to `system`.
- `system:part`: as always.

Saying that check my comments on the implementation later on.

Reply:
done